settings.gradle

```diff
@@ -7,4 +7,4 @@
  * in the user guide at https://docs.gradle.org/4.9/userguide/multi_project_builds.html
  */
 
-rootProject.name = 'http-server'
+rootProject.name = 'http-Server'
```

Comment:
I'm just curious.. why did this get changed?

Reply:
I'm also curious!

Reply:
ahh, those darn refactor tools. They really come in handy, but then sometimes they don't...